src/components/PicklistOption/index.js

```diff
@@ -19,39 +19,21 @@ class Option extends Component {
     }
 
     componentDidMount() {
-        const { privateRegisterChild, disabled, variant, label, name, icon, value } = this.props;
+        const { disabled, variant, name } = this.props;
         const isHeader = variant === 'header';
         if (disabled || isHeader || typeof name !== 'string') {
             return null;
         }
-        return setTimeout(
-            () =>
-                privateRegisterChild(this.itemRef.current, {
-                    label,
-                    name,
-                    icon,
-                    value,
-                }),
-            0,
-        );
+        return this.registerChild();
```

Comment:
sorry, let's call this method register since child here no make sense

Reply:
I also implemented an unregister method